src/Views/UserSettingsView.vala

```diff
@@ -487,13 +511,8 @@ namespace SwitchboardPlugUserAccounts.Widgets {
                 }
             }
 
-            var user_locked = user.get_locked ();
-            if (user_locked) {
-                enable_user_button.label = _("Enable Account");
-                enable_user_button.add_css_class (Granite.STYLE_CLASS_SUGGESTED_ACTION);
-            } else {
-                enable_user_button.label = _("Disable Account");
-                enable_user_button.remove_css_class (Granite.STYLE_CLASS_SUGGESTED_ACTION);
+            if (enable_action.state == user.get_locked ()) {
+                enable_action.set_state (!user.get_locked ());
```

Comment:
```suggestion
            var user_locked = user.get_locked ();
            if (enable_action.state == user_locked) {
                enable_action.set_state (!user_locked);
```

to avoid multiple function call that would return the same result